client/landing/stepper/declarative-flow/internals/steps-repository/choose-a-plan/style.scss

```diff
@@ -129,6 +129,26 @@ $videopress-mobile-layout-width: 700px;
 						}
 					}
 				}
+
+				.selected-plan-item {
+					background-color: #ffe61c !important;
+					color: #000 !important;
+					background-image: repeating-linear-gradient(
+						-45deg,
+						transparent,
+						transparent 1rem,
+						rgba(0, 0, 0, 0.1) 1rem,
+						rgba(0, 0, 0, 0.1) 2rem
+					) !important;
+					background-size: 200% 200% !important;
+					animation: videopressselectedplanitem 10s linear infinite !important;
+				}
+
+				@keyframes videopressselectedplanitem {
```

Comment:
nit: is there a way to make this name more readable? keyframe name supports `_` and `-`.